src/sql/base/browser/ui/table/media/slickColorTheme.css

```diff
@@ -122,6 +122,11 @@
 	background-image: url("viewChart.svg");
 }
 
+.vs .icon.viewSandDance {
```

Comment:
shouldn't this be from an extension?  Perhaps I missed what the planned feature is here.  Can we do a design review of what is happening here with the Query Editor committers?